README.md

```diff
@@ -30,10 +39,25 @@ Configuration options are documented in [Configuration.md](https://github.com/gi
 
 Before upgrading, please check [upgrading.md](https://github.com/giantswarm/cert-manager-app/blob/master/docs/upgrading.md).
 
+## Release Process
+
+* Ensure CHANGELOG.md is up to date.
+* Create a new branch to trigger the release workflow as either a patch, minor, or major. E.g. to release a patch, create a branch from master called master#release#patch and push it. Automation will create a release PR.
+* Merging the release PR will push a new git tag and trigger a new tarball to be pushed to the
+  [giantswarm-catalog].
+* Test and verify the cert-manager release across supported environments in a new or existing WIP platform release.
+
+## Contributing & Reporting Bugs
+If you have suggestions for how `cert-manager` could be improved, or want to report a bug, open an issue! We'd love all and any contributions.
+
+Check out the [Contributing Guide](https://github.com/giantswarm/cert-manager-app/blob/main/CONTRIBUTING.md) for details on the contribution workflow, submitting patches, and reporting bugs.
+
+
 ---
 
 [app-operator]: https://github.com/giantswarm/app-operator
 [cluster-operator]: https://github.com/giantswarm/cluster-operator
+[cert-manager]: https://github.com/jetstack/cert-manager
```

Comment:
Repo was moved to https://github.com/cert-manager/cert-manager 
GitHub redirects, but I think its worth having an up to date URL here